src/screens/MainHeader/index.js

```diff
@@ -8,6 +8,11 @@ import LogInTab from '../LogInTab';
 import SignUpTab from '../SignUpTap';
 
 class MainHeader extends Component {
+  handleLogOff = e => {
+    document.cookie = "user_social_cat=;expires=Thu, 01 Jan 1970 00:00:01 GMT;";
+    document.cookie = "token_social_cat=;expires=Thu, 01 Jan 1970 00:00:01 GMT;";
+    window.location.href = "/";
```

Comment:
Esto no deberías hacerlo con el router?

Reply:
No, el router se encarga de definir que componente renderea que ruta, al cambiar la ruta ya directamente sabe que componente renderear. La cagada de hacer esta llamada con `window.location.href` es que pierdo el state pero medio que me chupa un huevo que se pierda porque no estamos usando redux

Reply:
Para hacerlo sin que se pierda el state se tiene que llamar a algo como `this.props.dispatch(push("/"))` o algo por el estilo. Que es medio un overkill porque tengo que declarar esos elemens de redux y me parece al dope. Si alguna vez me interesa mantener el state de mi app lo voy a hacer asi

Reply:
```javascript
this.props.history.push('/')
```
 no sirve?